src/interaction/scroll.ts

```diff
@@ -19,7 +20,8 @@ export function start(): void {
 }
 
 export function observe(root: Node): void {
-    let node = root === document ? window : root;
+    let frame = iframe(root);
+    let node = frame ? frame.contentWindow : (root === document ? window : root);
```

Comment:
>contentWindow [](start = 29, length = 13)

Why contentWindow and not contentDocument?

Reply:
We have been tracking scroll events on the window from before (see red part). Explanation is in the same file below in comments:
  // Edge doesn't support scrollTop position on document.documentElement.
  // For cross browser compatibility, looking up pageYOffset on window if the scroll is on document.